worker/src/metrics.rs

```diff
@@ -0,0 +1,43 @@
+use sysinfo::System;
+
+pub enum MemoryMeasure {
+    Megabyte,
+    Gigabyte,
+}

```

Comment:
```suggestion
pub enum SpaceUnit {
    MiB,
    GiB,
}

```

Reply:
Also, I'm not sure about the "space" unit thing. Maybe `BitUnit`, or `InformationUnit`? idk...

https://en.wikipedia.org/wiki/Byte